PowerEditor/installer/nativeLang/spanish.xml

```diff
@@ -630,14 +630,15 @@ Translation note:
 					<Item id="2219" name="Palabras clave predeterminadas"/>
 					<Item id="2221" name="Palabras clave definidas por el usuario"/>
 					<Item id="2225" name="Lenguaje:"/>
-					<Item id="2226" name="Habilitar color global de fuentes"/>
-					<Item id="2227" name="Habilitar color global de fondo"/>
-					<Item id="2228" name="Habilitar tipo global de fuente"/>
-					<Item id="2229" name="Habilitar tamaño global de fuente"/>
-					<Item id="2230" name="Habilitar estilo global en negrita"/>
-					<Item id="2231" name="Habilitar estilo global en cursiva"/>
-					<Item id="2232" name="Habilitar estilo global en subrayado"/>
+					<Item id="2226" name="Forzar el color de primer plano para todos los estilos"/>
+					<Item id="2227" name="Forzar el color de fondo para todos los estilos"/>
+					<Item id="2228" name="Forzar la elección del tipo de letra para todos los estilos"/>
+					<Item id="2229" name="Forzar tamaño de fuente para todos los estilos"/>
+					<Item id="2230" name="Forzar negrita para todos los estilos"/>
+					<Item id="2231" name="Forzar cursiva para todos los estilos"/>
+					<Item id="2232" name="Forzar subrayado en todos los estilos"/>
 					<Item id="2234" name="Ir a Configuración"/>
+					<Item id="2235" name="¿Qué es la anulación global?"/>
```

Comment:
Don't translate "&quot;Global override&quot;

Reply:
Done. 
Please check line 1822: `<!-- Don't translate "&quot;Global override&quot; and &quot;Default Style&quot; -->`

Don't translate **" ??** ...